// Copyright 2020 The Chromium Authors
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

syntax = "proto3";

package feedwire;

import "components/feed/core/proto/v2/wire/consistency_token.proto";

option optimize_for = LITE_RUNTIME;

message UploadActionsResponse {
  ConsistencyToken consistency_token = 1;
}
